### ISSUE TITLE
debug to auth.log or other files possible if not locally called?

### ISSUE THREAD

**Issue by Reiner030** [Yubico/pam-u2f#50]
Is it possible to get the debug output also for ssh sessions to e.g. auth.log ?

In my first tests the individual mapping files were ignored/not found - no idea what happend there.
I had to use a local "login" test with debug activated to see that my key copy was somehow not correct so I got this error from key:

````util.c:            D(("Length of key number %d not even", i + 1));````

Visually I saw no error/difference but a new copy&paste entry fixed it.

**Comment by a-dma** [comment]
Could you please elaborate a bit more on the use case? How are you using the module and where/when would you expect the debug info?

**Comment by Reiner030** [comment]
I tried (finally successfull) to use U2F in my developer VM to get SSH access with 2 separate keys.

Somehow there was first a copy/paste problem (newline / unprinted character?) which I didn't catched and I was thinking that the pam module ignored my authfile parameter when I set it to an individual mapping file.
Only after testing it on console in my VM I saw the debug messages on "stdout" and so I was notified about the key length problem.

Therefore it would be nice if errors / debug messages could be logged to auth.log/syslog daemon..
